project/bootstrap/github_deploy_role_cfn.yml

```diff
@@ -0,0 +1,138 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Role for Deploy runner"
+
+Parameters:
+  DeployRoleName:
+    Type: String
+    Default: appFilmDropDeployRole
+  DeployRolePolicyName:
+    Type: String
+    Default: appFilmDropDeployPolicy
+
+Resources:
+  DeployRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: !Ref DeployRoleName
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Federated:
+                - !Sub "arn:aws:iam::${AWS::AccountId}:oidc-provider/token.actions.githubusercontent.com"
+            Action:
+              - "sts:AssumeRoleWithWebIdentity"
+            Condition:
+              StringLike:
+                "token.actions.githubusercontent.com:sub": "repo:RepoOrg/RepoName:*"
+              "ForAllValues:StringEquals":
+                "token.actions.githubusercontent.com:iss": "https://token.actions.githubusercontent.com"
+                "token.actions.githubusercontent.com:aud": "sts.amazonaws.com"
+  DeployRolePolicy:
```

Comment:
I'll think on this a bit, I feel we should have here just the definition of the deploy Policy and not the role with the trust policy. 🤔